Lib/idlelib/pyshell.py

```diff
@@ -1698,6 +1698,8 @@ def main():
         root.mainloop()
     root.destroy()
     capture_warnings(False)
+    import _tkinter
+    _tkinter.finalize_tcl()
```

Comment:
```suggestion
    _tkinter._finalize_tcl()
```
so we can backport to 3.9?

Reply:
Done.